scripts/prepare-coverage.bat

```diff
@@ -0,0 +1,7 @@
+if not defined PYTEST_NO_COVERAGE (
+    set "_PYTEST_TOX_COVERAGE_RUN=coverage run --source {envsitepackagesdir}/_pytest/,%CD%/testing -m"
```

Comment:
We could use here `{envinidir}` instead of `%CD%` probably.. (also with Travis then)

Reply:
Oh good catch. Will do.